assets/javascripts/discourse/widgets/discourse-reactions-state-panel-reaction.js.es6

```diff
@@ -42,15 +51,10 @@ export default createWidget("discourse-reactions-state-panel-reaction", {
 
     if (attrs.users.length > MIN_USERS_COUNT) {
       list.push(
-        this.attach("button", {
-          action: "showUsers",
-          contents: [
-            iconNode(attrs.isDisplayed ? "chevron-up" : "chevron-down"),
-          ],
-          actionParam: attrs,
-          className: "show-users",
-          title: "",
-        })
+        h(
```

Comment:
note we should implement bubbles=false into core widgets will make a PR about this

Reply:
hmmm actually I see ember deprecated bubbles... so I guess it's fine like this